### ISSUE TITLE
domainID calculated by CERTBOT_DOMAIN

### ISSUE THREAD

**Issue by ckhmer1** [aney1/certbot-domainvalidation-dynu#1]
Hi,
thanks for your great job.

The scripts are working fine but only works for one domain.
What about if there are more domains?

Just replace the 
domainID='<PUT_YOUR_DOMAIN_ID_HERE>'
with

dns=\$(curl -X GET https://api.dynu.com/v2/dns -H "accept: application/json" -H "API-Key: \$api_key")
domainID=\$(echo \$dns  | jq ".domains[] | select(.name==\"\$CERTBOT_DOMAIN\")" | jq '.id')

Thanks

**Comment by aney1** [comment]
This code snippet doesn't work for me.
After a long time of playing around I finally found out what's wrong. (I'm a jq noob)
The quotation marks have to be escaped:
```
dns=$(curl -X GET https://api.dynu.com/v2/dns -H "accept: application/json" -H "API-Key: $api_key")
domainID=$(echo $dns | jq ".domains[] | select(.name==\"$CERTBOT_DOMAIN\")" | jq '.id')
```

Also sorry for the late response, but github doesn't notify me for new issues. I only found this issue by chance, because of a pull request.

**Comment by aney1** [comment]
Merged: 5d18c6fca72ed68a7f1e655f16f1dcfac7739d88
